ingest-app/docs/api.md

```diff
@@ -22,6 +22,9 @@ Join the [CMR Client Developer Forum](https://wiki.earthdata.nasa.gov/display/CM
   * /providers/\<provider-id>/services/\<native-id>
```

Comment:
This is not just related to your change. Can you update the Content-Type section of the API doc to put in the correct content-types for UMM-JSON format? We are missing variable, service, subscription and tool.